cmd/generate.go

```diff
@@ -28,155 +28,165 @@ var generateCmd = &cobra.Command{
 	Short: "Generate a config file from state management",
 	Run: func(cmd *cobra.Command, args []string) {
 		// make sure that we have a token present before trying to make request
-		if config.AccessToken == "" {
+		if conf.AccessToken == "" {
 			// check if OCHAMI_ACCESS_TOKEN env var is set if no access token is provided and use that instead
 			accessToken := os.Getenv("ACCESS_TOKEN")
 			if accessToken != "" {
-				config.AccessToken = accessToken
+				conf.AccessToken = accessToken
 			} else {
 				// TODO: try and fetch token first if it is needed
 				if verbose {
-					fmt.Printf("No token found. Attempting to generate config without one...\n")
+					fmt.Printf("No token found. Attempting to generate conf without one...\n")
 				}
 			}
 		}
 
 		// use cert path from cobra if empty
-		if config.CertPath == "" {
-			config.CertPath = cacertPath
+		if conf.CertPath == "" {
+			conf.CertPath = cacertPath
 		}
 
-		// show config as JSON and generators if verbose
+		// show conf as JSON and generators if verbose
 		if verbose {
-			b, err := json.MarshalIndent(config, "", "  ")
+			b, err := json.MarshalIndent(conf, "", "  ")
 			if err != nil {
-				fmt.Printf("failed to marshal config: %v\n", err)
+				fmt.Printf("failed to marshal conf: %v\n", err)
```

Comment:
If we are going to print out log/error messages and also print the config to stdout when the output file path is not specified (below), then we should definitely send these messages to stderr.

Reply:
This should actually be using `log.Error()` instead, which should fix the issue since it should be set to print to stderr.